README.md

```diff
@@ -116,9 +116,13 @@ From root repo directory:
 
 After running through the config setup UI flow once, you will need to delete the `fedimintd` data to run through it again. To do this, delete the `fm_1`, `fm_2`, `fm_3`, and `fm_4` folder from the repo. These are data directories mounted to Docker containers running fedmintd and are listed in `.gitignore` so are safe to remove.
 
-### Running with mprocs
+### Running with Nix and mprocs
 
 1. Install [mprocs](https://github.com/pvolok/mprocs)
+1. Install [Nix](https://nixos.org/download)
```

Comment:
we also need to add the determinate systems downloader ( here as recommendation for macOS, normal nix download causes lots of problems whenever macOS updates 

`curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix | sh -s -- install`

Reply:
Roger this. Will do.